src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -133,7 +133,7 @@ void EnMag_Init(Actor* thisx, PlayState* play) {
     this->mainTitleAlpha = this->subtitleAlpha = this->unk11F32 = this->copyrightAlpha = 0;
 
     this->unk11F02 = 30;
-    this->unk11F00 = this->state = MAG_STATE_INITIAL;
+    this->ocarinaStaff = this->state = MAG_STATE_INITIAL;
```

Comment:
Huh, seems to have been named by mistake. Both `unk11F00` from the `MessageContext` Struct and `unk11F00` from the `EnMag` struct must have been caught by a global search and replace. I would have thought `unk11F00` would have been a unique name since structs normally don't go that high. The use in `EnMag` can be reverted